### ISSUE TITLE
`rocketpool service terminate` does not delete validator/wallet keys nor passwords

### ISSUE THREAD

**Issue by jshufro** [rocket-pool/smartnode#316]
Two minipools were slashed today with the following timeline

- NO ran `rocketpool service terminate` expecting this to delete his keys (it does not, currently, delete the keys)
- NO ran `rocketpool wallet purge` to be safe, but it failed due to the previous terminate, and they did not notice the error
- NO proceeded to restore wallet on a different node and eventually restarted the service on the original node, leading to attestation violations

https://github.com/rocket-pool/smartnode/pull/315 adds verbosity to the error reporting for the `purge` command, but we should have `terminate` do a best-effort deletion of the data dir for the project (default ~/.rocketpool/data). Right now, it simply runs `docker compose down`, which leaves the keys dangling, and prevents `purge` from being effective.

`terminate` should require elevated permissions (sudo, doas), and should remove the contents of ~/.rocketpool/data

We should eventually improve public-facing documentation with a clear and thorough guide on decommissioning a node's secrets.

Further, the `rocketpool service help` output for the `terminate` subcommand is currently misleading:
```
   terminate, t               Deletes all of the Rocket Pool Docker containers and volumes, including your ETH1 and ETH2 chain data and your Prometheus database (if metrics are enabled). Only use this if you are cleaning up the Smartnode and want to start over!
```

Terminate _currently_ just stops the containers from running and doesn't delete anything. We should update this text to accurately reflect the behavior, before and/or after `terminate` itself is adjusted.

**Comment by superphiz** [comment]
This seems to be a roughly similar occurrence to the one described in this issue: https://github.com/rocket-pool/smartnode/issues/212

**Comment by jclapis** [comment]
Resolved these in 096bbf168ac50e2639ee156822c72db44822d62b and 9b3ddcb7a21d3fe698ad8d235e52084d9cc98b14.
